project.php

```diff
@@ -20,10 +23,15 @@
 $project = $projectsModel->getProjectById($idLink);
 $projectTitle = ProjectDisplayService::displayProject($project);
 $projectDeadline = DateService::reformatDateUK($project->deadline);
+$projectDeadlineUS = DateService::reformatDateUS($project->deadline);
+if ($pageLocale == 'us') {
+    $projectDeadline = $projectDeadlineUS;
+}
```

Comment:
If the `$projectDeadlineUS` variable isn't being used anywhere else, get rid of line 26 and just have:

```
if ($pageLocale == 'us') {
    $projectDeadline = DateService::reformatDateUS($project->deadline);
}
```

Reply:
Also, make sure to change `==` to `===` wherever you can, it's a good habit to use `===` as much as possible